test_trajectory:
  ros__parameters:
    controller_name: "position_trajectory_controller"
    wait_sec_between_publish: 3
    repeat_the_same_goal: 1 # useful to simulate continuous inputs

    # goal_time_from_start: 3.0
    goal_names: ["pos1", "pos2", "pos3", "pos4"]
    pos1:
      positions: [0.785, 0.785, 0.785, 0.785, 1.0]
    pos2:
      positions: [0.0, 0.0, 0.0, 0.0, 0.0]
    pos3:
      positions: [-0.785, -0.785, 0.0, 0.0, 0.875]
    pos4:
      positions: [0.0, 0.0, 0.0, 1.0, 0.0, 0.875]

    joints:
      - base_to_shoulder_joint
      - shoulder_to_arm_joint
      - arm_to_elbow_joint
      - elbow_to_forearm_joint
      - forearm_to_wrist_joint
